source/posts/our-commitment-to-posi.html.md

```diff
@@ -0,0 +1,216 @@
+
+Goal to generate surplus – organisations which define sustainability based merely on recovering costs are brittle and stagnant. It is not enough to merely survive, it has to be able to adapt and change. To weather economic, social and technological volatility, they need financial resources beyond immediate operating costs.
+
+DataCite operates on a cost-recovery membership model. As our community continues to grow and we streamline our operations, we seek to generate a surplus each year. The General Assembly aims to continue to grow the reserve funds to support continued operations and pre-finance future organizational projects.
+
+Goal to create contingency fund to support operations for 12 months – a high priority should be generating a contingency fund that can support a complete, orderly wind down (12 months in most cases). This fund should be separate from those allocated to covering operating risk and investment in development.
+
+DataCite currently maintains an operational reserve in accordance with statutory association laws of Germany. The DataCite General Assembly is required to approve the use of reserves in accordance with the activities of the association. As such, we maintain a steady approach to continue to build a contingency fund that supports the ongoing charitable mission of the organization.
+
+Mission-consistent revenue generation – potential revenue sources should be considered for consistency with the organisational mission and not run counter to the aims of the organisation. 
+
+DataCite revenue is generated directly through membership, use of member services and funded project activities. All revenue streams are directly linked to the organizational mission.
+
+Revenue based on services, not data – data related to the running of the research enterprise should be a community property. Appropriate revenue sources might include value-added services, consulting, API Service Level Agreements or membership fees.
+
+DataCite offers a portfolio of member services and has no intention to change our strategy in this regard. Our services are grouped within three distinct categories, namely; [discovery](https://commons.datacite.org/), [registration](https://doi.datacite.org/) and [integration](https://datacite.org/integratorapis.html). We provide the means to create, find, cite, connect, and use research.
+
+
+## Insurance
+
+
+<table>
```

Comment:
@mjbuys I think you will have to convert those tables with https://jmalarcon.github.io/markdowntables/